### ISSUE TITLE
DIC does not check if the destination has enough space for a dump

### ISSUE THREAD

**Issue by InternalLoss** [saramibreak/DiscImageCreator#83]
**Describe the bug**
This was discussed in VGPC - It's possible for DIC to run out of space on a hard drive, and continue dumping without warning the user, and will even output (incorrect) hashes.

Functionality to check for this was added to MPF - https://github.com/SabreTools/MPF/issues/299 - but this should be part of DIC itself as users will inevitably use DIC on its own and could submit bad dumps without knowing.

**Comment by gmipf** [comment]
I also already noticed that behavior. But for me the best practice is to always dump new discs at least twice and compare the hashes. I got multiple times bad dumps without DIC or the drive even noticing it (no c2 error etc.), in that case I had to dump a 3rd time. This is mostly because of bad interface connection or other interface errors but can also be caused by drive cache errors I think. So don't trust the error detection of the drive.
